corehq/toggles.py

```diff
@@ -855,7 +855,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
 )
 
 
-def enable_toggles_for_scale_beta(domain):
-    USE_SQL_BACKEND.set(domain, True, namespace=NAMESPACE_DOMAIN)
+def enable_toggles_for_scale(domain):
     NEW_EXPORTS.set(domain, True, namespace=NAMESPACE_DOMAIN)
     TF_USES_SQLITE_BACKEND.set(domain, True, namespace=NAMESPACE_DOMAIN)
```

Comment:
Not thrilled about adding all new domains to these toggles but I don't see much option at this stage.